eole/models/model.py

```diff
@@ -974,13 +974,12 @@ def embed_vision_language_features(self, src, images):
 
         return combined_features
 
-    def forward(self, src, tgt, src_len, bptt=False, with_align=False, images=[]):
+    def forward(self, src, tgt, src_len, with_align=False, **kwargs):
         """A DecoderModel forward the src side to the decoder along
         with the source lengths vector. It is a decoder only LM (cf GPT-2)"""
 
-        if not bptt:
-            self.decoder.init_state()
-        emb = self.embed_vision_language_features(src, images)
+        self.decoder.init_state()
+        emb = self.embed_vision_language_features(src, **kwargs)
```

Comment:
I guess this works if `images` is the first (and only?) item in `kwargs`, but it would probably be cleaner to adapt `embed_vision_language_features` to grab it explicitly from the kwargs dict.